src/routes/(app)/+page.svelte

```diff
@@ -57,8 +58,9 @@
 
   // Use if not initializing journey in a "context module"
   onMount(async () => {
-    if (suspendedIdParam) {
+    if (suspendedIdParam || formPostEntryParam) {
       journeyStore.resume(location.href);
+      history.replaceState({}, '', '/');
```

Comment:
I noticed that we are not sanitizing the URL after resuming the flow, whether from Suspended or Social login, so decided to look at how we can mutate the query parameters using Svelte Kit. My findings soon showed that this is not possible (or at least doesn't seem to be possible), so instead I invoked the browser API to rewrite the current URL. @cerebrl as a Svelte ninja there may well be a better way you know of to do this, so please share your thoughts!

Reply:
I think the SvelteKit API you're looking for is the `goto` method found in `$app/navigation`:

```
goto('/', { replaceState: true });
```

Docs: https://kit.svelte.dev/docs/modules#$app-navigation-goto

Reply:
Ah I must have missed that. Perfect, thanks.